source/common/access_log/access_log_manager_impl.cc

```diff
@@ -20,8 +20,8 @@ AccessLogManagerImpl::~AccessLogManagerImpl() {
 }
 
 void AccessLogManagerImpl::reopen() {
-  for (auto& [log_key, log_file_ptr] : access_logs_) {
```

Comment:
nit: I think you can use `_` as the 2nd one to keep structured binding? Does that work?

Reply:
I looked for something like that (or `std::ignore`) but it seemed that wasn't available for structured bindings, I can do some more digging/experimenting and get back to you

Reply:
I saw someone do this in a recent PR, but maybe I'm making that up.

Reply:
According to Google I'm making that up. Oh well!

Reply:
According to http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0144r2.pdf (section 3.8) there is no way to ignore an element, `_` compiles but it is a valid variable name

Reply:
Yeah, I wanted to do `_` as well b/c of becoming familiar with Googletest/mock